lib/linter/linter.js

```diff
@@ -1712,10 +1679,13 @@ class Linter {
                 t = startTime();
             }
 
-            const parseResult = parse(
+            const parserService = new ParserService();
+            const parseResult = parserService.parseSync(
                 file,
-                config.language,
-                languageOptions
+                {
+                    ...config,
+                    languageOptions
+                }
```

Comment:
```suggestion
                config
```

I believe this can be simplified this way.